base_layer/core/src/transactions/helpers.rs

```diff
@@ -332,8 +331,7 @@ pub fn spend_utxos(schema: TransactionSchema) -> (Transaction, Vec<UnblindedOutp
     };
     outputs.push(change_output);
     match stx_protocol.finalize(KernelFeatures::empty(), &factories) {
-        Ok(true) => (),
-        Ok(false) => panic!("{:?}", stx_protocol.failure_reason()),
+        Ok(_0) => (),
         Err(e) => panic!("{:?}", e),
```

Comment:
I know its outside of your changes but a panic here seems a bit drastic? Shouldn't this error be propagated

Reply:
This helper function is only used in tests, so maybe all good?